src/objects/zcl_abapgit_oo_deserializer.clas.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
```

Comment:
is this a new class or something renamed?

Reply:
ZCL_ABAPGIT_OO_DESERIALIZER is a new class containing moved methods. Not related to the issue but helps organizing and simplifying for solving the issue. I propose to split in 2 PRs.